app/model/user/User.java

```diff
@@ -62,4 +65,15 @@ public void setEmailAddress(String emailAddress)
 	{
 		this.emailAddress = emailAddress;
 	}
+	
+	public String getAvatarURL()
+	{
+		return avatarURL;
+	}
+	
+	public String setAvatarURL(String userName)
+	{
+		/* Below URL is of the Crucible ( codereview ) server */
+		this.avatarURL = "http://srv-ind-scrat.vanenburg.com:8060/avatar/" + userName;
```

Comment:
Shouldn't the ?s=number be part of this code as well? That's make it trivial to switch to another avatar source (like Jira).